pkg/artifacts/signable.go

```diff
@@ -88,13 +88,6 @@ type StructuredSignable struct {
 
 func (oa *OCIArtifact) ExtractObjects(tr *v1beta1.TaskRun) []interface{} {
 	imageResourceNames := map[string]*image{}
-	if tr.Status.TaskSpec != nil && tr.Status.TaskSpec.Resources != nil {
```

Comment:
We should also remove this in `func materials(tr *v1beta1.TaskRun)` in which resource input params will overwrite others.

Reply:
That's a good catch.